scripts/run_integration_tests.sh

```diff
@@ -157,7 +155,7 @@ cp target/debug/cloud-hypervisor $VFIO_DIR
 sudo adduser $USER kvm
 newgrp kvm << EOF
 export RUST_BACKTRACE=1
-cargo test --features "integration_tests" -- --nocapture
+cargo test vhost_user_blk --features "integration_tests" -- --nocapture
```

Comment:
You need to remove that change.

Reply:
But this way is way faster! xD
Thanks for pointing to this, I've just removed the change.